.github/index.pkl

```diff
@@ -0,0 +1,68 @@
+//===----------------------------------------------------------------------===//
+// Copyright © 2025 Apple Inc. and the Pkl project authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//===----------------------------------------------------------------------===//
+
+amends "@pkl.impl.ghactions/PklCI.pkl"
+
+import "@gha/actions/Common.pkl"
+import "@gha/actions/Setup.pkl"
+import "@gha/Workflow.pkl"
+import "@pkl.impl.ghactions/jobs/HawkeyeCheck.pkl"
+import "@pkl.impl.ghactions/steps/SetupPkl.pkl"
+
+local test: Workflow = new {
+  jobs {
+    ["test"] {
+      `runs-on` = "ubuntu-latest"
+      steps {
+        new Common.Checkout {}
+        new Setup.Go {
+          with {
+            `go-version` = "1.25"
```

Comment:
```suggestion
            `go-version` = "1.25"
            `check-latest` = true
            `cache-dependency-path` = "go.work.sum"
```

`check-latest` asks setup-go to find the latest patch for the given release. Otherwise, it just installs the latest version in the cache.